aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -52,3 +52,13 @@ proposals:
               sender_aware_v2: 32
             block_gas_limit: 35000
             transaction_deduper_type: txn_hash_and_authenticator_v1
+   - name: enable_bulletproofs
```

Comment:
YAML is horrible, I think you have extra space on this line

Reply:
Correct. I think it is now fixed